### DISCUSSION TITLE
Win11 blocks OpenshellSetup_4_4_194.exe

### DISCUSSION THREAD

**Discussion by tajemniczy22** [Open-Shell/Open-Shell-Menu#2082] [Q&A] [answered]
when i try to open openshell my win defender blocks it and says taht it can be a threat

**Comment by ge0rdi** [comment, marked as answer]
This is a false-positive. There is no _threat_ in the installer (if you downloaded it here from github).

**Comment by tajemniczy22** [comment]
Thanks for answering my question but i still have some left. How do i
unistal it if i dont want it anymore? Could it break my desktop?

**Reply by ge0rdi** [reply]
> How do i unistal it if i dont want it anymore?

Just like any other application.

**Reply by tajemniczy22** [reply]
what about my other questions?

**Reply by Ibuprophen** [reply]
I've never encountered a problem when Uninstalling it as I would with any other software. 

~Ibuprophen

**Reply by tajemniczy22** [reply]
what about bugs? COuld they damage my computer? Like flashing icons or something similar?

**Reply by Ibuprophen** [reply]
I'm not sure what your specifically referring to and how it's tied specifically to Open-Shell, but I'm going to guess what you're asking about. 

Uninstalling Open-Shell doesn't specifically affect icons because what icons you're seeing in the menu is just what's in those specific applicable directories. Open-Shell doesn't affect icons specifically and Uninstalling it won't affect icons. 

I hope I had explained the above okay via text... 👍 

~Ibuprophen

**Reply by tajemniczy22** [reply]
i don't mean uninstaling open shell but when i instal it could it have bugs that would impair your computer use.

**Reply by Ibuprophen** [reply]
I've never personally experienced any issues, but I can only attest for my 2 PCs and 3 Laptops. As with any software, anyone can experience something different depending upon their own personal system setup. 

My opinion would be that most individuals doesn't experience any negative affects tied specifically to Open-Shell. 

~Ibuprophen
